tests/Provider/ImageProviderTest.php

```diff
@@ -122,6 +122,9 @@ public function testHelperProperties()
                 $adminBox, // Third call
                 $largeBox, // Fourth call
                 $mediumBox,
+                $largeBox,
+                $largeBox, // Fifth call
```

Comment:
LargeBox twice is correct?

Reply:
yes, processing order is format parameter followed by a loop over the picture array. For the test cases I added the sequence is large-medium-large, large-medium-large